back/src/schema.ts

```diff
@@ -18,10 +18,19 @@ export const typeDefs = gql`
         passport: Int!
     }
 
+    input getViolationInput {
```

Comment:
assim como a regra de nomeclatura para classes e interfaces, os types e inputs do graphql são iniciados com letras maiusculas.

  ```suggestion
    input GetViolationInput {
```

O mesmo para o `getTravellerInput`